platform/ios/platform/ios/app/MBXViewController.m

```diff
@@ -1898,56 +1898,53 @@ - (IBAction)handleLongPress:(UILongPressGestureRecognizer *)longPress
     }
 }
 
-- (IBAction)cycleStyles:(__unused id)sender
+// MARK: - User defined Styles
+- (void)setStyles
 {
-    static NSArray *styleNames;
-    static NSArray *styleURLs;
-
-    static dispatch_once_t onceToken;
-    dispatch_once(&onceToken, ^{
-        styleNames = @[
-            @"Streets",
-            @"Outdoors",
-            @"Light",
-            @"Dark",
-            @"Satellite",
-            @"Satellite Streets",
-        ];
-        styleURLs = @[
-            [MGLStyle streetsStyleURL],
-            [MGLStyle outdoorsStyleURL],
-            [MGLStyle lightStyleURL],
-            [MGLStyle darkStyleURL],
-            [MGLStyle satelliteStyleURL],
-            [MGLStyle satelliteStreetsStyleURL]
-        ];
-        NSAssert(styleNames.count == styleURLs.count, @"Style names and URLs don’t match.");
-
-        // Make sure defaultStyleURLs is up-to-date.
-        unsigned numMethods = 0;
-        Method *methods = class_copyMethodList(object_getClass([MGLStyle class]), &numMethods);
-        unsigned numStyleURLMethods = 0;
-        for (NSUInteger i = 0; i < numMethods; i++) {
-            Method method = methods[i];
-            if (method_getNumberOfArguments(method) == 3 /* _cmd, self, version */) {
-                SEL selector = method_getName(method);
-                NSString *name = @(sel_getName(selector));
-                if ([name hasSuffix:@"StyleURLWithVersion:"]) {
-                    numStyleURLMethods += 1;
-                }
-            }
-        }
-        NSAssert(numStyleURLMethods == styleNames.count,
-                 @"MGLStyle provides %u default styles but iosapp only knows about %lu of them.",
-                 numStyleURLMethods, (unsigned long)styleNames.count);
-    });
+    self.styleNames = [NSMutableArray array];
+    self.styleURLs = [NSMutableArray array];
+    
+    /// Style that does not require an `accessToken` nor any further configuration
+    [self.styleNames addObject:@"Zeroconf Style"];
+    [self.styleURLs addObject:[NSURL URLWithString:@"https://raw.githubusercontent.com/roblabs/openmaptiles-ios-demo/master/OSM2VectorTiles/styles/geography-class.GitHub.json"]];
+
+    /// Add Mapbox Styles if an `accessToken` exists
+    if ([MGLAccountManager accessToken].length)
```

Comment:
same as above

Reply:
We should remove the check for `accessToken`, as well as there is a *Run Script* that inserts the token that should also be removed.  I left it in there in case MapTiler team wanted to modify, rename and reuse that script.